merlin/systems/dag/ops/feast.py

```diff
@@ -19,8 +20,48 @@
 
 
 class QueryFeast(PipelineableInferenceOperator):
+    """
+    The QueryFeast operator is responsible for ensuring that your feast feature store [1]
+    can communicate correctly with tritonserver for the ensemble feast feature look ups.
+
+    References
+    ----------
+    [1] https://docs.feast.dev/
+    """
+
     @classmethod
-    def from_feature_view(cls, store, path, view, column, output_prefix=None, include_id=False):
+    def from_feature_view(
+        cls,
+        store: FeatureStore,
+        path: str,
+        view: str,
+        column: str,
+        output_prefix: str = None,
+        include_id: bool = False,
+    ):
+        """
+        Allows for the creation of a QueryFeast operator from already created Feast artifacts.
+
+        Parameters
+        ----------
+        store : FeatureStore
+            Previously loaded feast feature store.
+        path : str
+            Path to the feast feature repo.
+        view : str
+            The feature view you want to pull feature from.
+        column : str
+            The column that input data will match against.
```

Comment:
If this desc is obvious and clear to the intended audience, leave it.  Because this is new to me, I can't tell in what way the values need to match.  If there's a little more to say, please add. Otherwise, I prolly shouldn't be so ignorant of Feast.

Please add ending punctuation to the next two.

For `include_id`, maybe "A boolean that specifies whether to include..."